CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [2.2.0]
+
+### Added
+
+- Added `/filesystem/{system_name}/transfer/compress` and `/filesystem/{system_name}/transfer/extract`
+  - `compress` operations (on `transfer` and `ops` endpoints) accept `pattern` parameter to compress files using `regex` syntax.
```

Comment:
@jpdorsch you should update the name here as well

Reply:
Done